requirements.txt

```diff
@@ -1,6 +1,6 @@
 sphinx == 3.5.0
```

Comment:
@ravindk89 Looks like extension `myst_parser` is missing from `requirements.txt`, leading to an error while building the HTML.

Reply:
```
Extension error:
Could not import extension myst_parser (exception: No module named 'myst_parser')
```

Reply:
I cleaned out my local venv, fixed requirements.txt, and tried again - error appears to be cleared out.

Reply:
@rushenn LMK if you still run into trouble.